src/components/list/list-item.types.ts

```diff
@@ -1,4 +1,4 @@
-import { MenuItem } from '@limetech/lime-elements';
+import { MenuItem } from '../menu/menu.types';
```

Comment:
This was what was preventing the _Types_ section of the docs from working correctly.

I'm guessing importing from `@limetech/lime-elements` doesn't work in `.types.ts` files, because the types haven't finished generating and thus haven't been linked up to the `@limetech/lime-elements` "alias" yet…? Importing from `@limetech/lime-elements` works in other parts of the code, like in `menu.tsx`.